app/assets/javascripts/votes.js

```diff
@@ -0,0 +1,73 @@
+// $('#small').id = "big";
+// $('#big').id = "small";
+// $('#small').append("hi");
+
+$(document).ready(function() {
+  $(".voting").on("click", 'span', function(event) {
+    event.preventDefault();
+    var userId = $('.username').attr('id')
```

Comment:
As far as I can tell you're not using the `user_id` in your controller, so remove this.